lib/llvmopencl/LLVMUtils.h

```diff
@@ -108,4 +110,28 @@ void setFuncArgAddressSpaceMD(llvm::Function *Func, unsigned ArgIndex,
 llvm::Metadata *createConstantIntMD(llvm::LLVMContext &C, int32_t Val);
 }
 
+template <typename VectorT>
+void CloneFunctionIntoAbs(llvm::Function *NewFunc,
+                          const llvm::Function *OldFunc,
+                          llvm::ValueToValueMapTy &VMap, VectorT &Returns,
+                          bool sameModule = true,
+                          const char *NameSuffix = "",
+                          llvm::ClonedCodeInfo *CodeInfo = nullptr,
+                          llvm::ValueMapTypeRemapper *TypeMapper = nullptr,
+                          llvm::ValueMaterializer *Materializer = nullptr) {
+
+
+  CloneFunctionInto(NewFunc, OldFunc, VMap,
+#ifdef LLVM_OLDER_THAN_13_0
```

Comment:
Better just duplicate the call for the ifdef branches for impoved readability.

Reply:
done